packtools/sps/validation/front_articlemeta_issue.py

```diff
@@ -228,4 +254,92 @@ def validate(self, data):
         vol_results.update(issue_results)
         vol_results.update(suppl_results)
         
-        return vol_results
\ No newline at end of file
+        return vol_results
+
+
+def _response(sub_item, expected, advice, obtained=None):
+    return format_response(
+        title='Pagination validation',
+        parent=None,
+        parent_id=None,
+        item='article-meta',
+        sub_item=sub_item,
+        validation_type='exist',
+        is_valid=False,
+        expected=expected,
+        obtained=obtained,
+        advice=advice
+    )
+
+
+class Pagination:
+    def __init__(self, xml_tree):
+        self.xml_tree = xml_tree
+        self.issue = ArticleMetaIssue(xml_tree)
+
+    def validation_pagination_attributes_exist(self):
+        """
+        Checks for the existence of starting and ending page numbers that cannot coexist with the elocation-id.
+
+        XML input
+        ---------
+        <article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article" xml:lang="en">
+            <front>
+                <article-meta>
+                    <lpage>240</lpage>
+                </article-meta>
+            </front>
+        </article>
+
+        Returns
+        -------
+        list of dict
+            A list of dictionaries, such as:
+            [
+                {
+                    'title': 'Pagination validation',
+                    'parent': None,
+                    'parent_id': None,
+                    'item': 'article-meta',
+                    'sub_item': 'elocation-id',
+                    'response': 'ERROR',
+                    'expected_value': 'no values for fpage and lpage OR no value for elocation-id',
+                    'got_value': 'elocation-id: e51467, fpage: 220, lpage: 240',
+                    'message': 'Got elocation-id: e51467, fpage: 220, lpage: 240, expected no values for fpage and lpage '
+                               'OR no value for elocation-id',
+                    'validation_type': 'exist',
+                    'advice': 'remove values for fpage and lpage OR remove value for elocation-id',
+                    'data': {'elocation_id': 'e51467', 'fpage': '220', 'lpage': '240'}
+                },...
+            ]
+        """
+        if self.issue.elocation_id is None:
```

Comment:
@Rossi-Luciano a lógica não está errada mas pode ser melhor. Error se todos fpage, lpage e elocation-id existem E se todos ausentes (**exceto se é o volume e number is None**). Fora isso, deve apresentar os dados encontrados.

O que tem que melhorar na sua versão é que por não encontrar elocation-id, você está solicitando fornecer fpage e lpage, mas e se o caso é de ter o elocation-id? Então, na verdade, a mensagem teria que ser que não foi encontrado nenhum que e `é necessário fornecer ou o fpage + lpage ou o elocation-id`.

O advice para os dois casos de **error** deve ser: `é necessário fornecer ou o fpage + lpage ou o elocation-id`.
E para todos os casos ERROR e OK, tem que mostrar o obtido